src/main/java/com/microsoft/azure/management/cosmosdb/samples/CreateCosmosDBWithEventualConsistency.java

```diff
@@ -106,34 +106,28 @@ public static boolean runSample(Azure azure, String clientId) {
         return false;
     }
 
-    private static void createDBAndAddCollection(String masterKey, String endPoint) throws DocumentClientException {
+    private static void createDBAndAddCollection(String masterKey, String endPoint) throws CosmosException {
```

Comment:
We have rebranded Cosmos names. Like `Collection` to `Container` - please change it here.